make/dev/nodeclarity/entrypoint.sh

```diff
@@ -19,7 +19,17 @@ if [ ! -z "$npm_proxy" -a "$npm_proxy" != " " ]; then
 fi
 
 npm install
-ng build
+
+./node_modules/.bin/ngc -p tsconfig-aot.json
+sed -i 's/* as//g' src/app/shared/gauge/gauge.component.js
```

Comment:
Should raise a new issue to remove the jquery dept and use native Angular funtions